CHANGES.md

```diff
@@ -1,5 +1,14 @@
 # Changes
 
+## edge-20.7.4
+
+This edge release brings support for EndpointSlices.
```

Comment:
It might be nice to have a little more discussion of what endpoint slices are, what this change means for the user, etc. Maybe:
```suggestion
This edge release adds support for the new Kubernetes [EndpointSlice] resource to
the Destination controller. Using the EndpointSlice API is more efficient for the 
Kubernetes control plane than using the Endpoints API. If the cluster supports 
EndpointSlices (a beta feature in Kubernetes 1.17), Linkerd can be installed with 
```console
linkerd install --enable-endpoint-slices
```
to use this resource rather than the Endpoints API.

[EndpointSlice](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/)
```